pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -98,15 +98,21 @@ func (plugin *awsElasticBlockStorePlugin) SupportsBulkVolumeVerification() bool
 }
 
 func (plugin *awsElasticBlockStorePlugin) GetVolumeLimits() (map[string]int64, error) {
+	volumeLimits := map[string]int64{
+		util.EBSVolumeLimitKey: 39,
+	}
 	cloud := plugin.host.GetCloudProvider()
 
+	// if we can't fetch cloudprovider we return a limit assuming
+	// kubelet could be running without cloudprovider.
+	if cloud == nil {
+		return volumeLimits, nil
```

Comment:
In the hindsight, this may not be best idea. Not returning a value from here is probably better because then an external CCM or admin can override it.